tests/test_cliCmdTestIngest.py

```diff
@@ -43,6 +44,9 @@ def makeExpectedKwargs(**kwargs):
 
 class Suite(unittest.TestCase):
 
+    def setUp(self):
+        sys.modules["lsst.obs.base.script.ingestRaws"] = lsst.obs.base.cli.test.ingestRaws
```

Comment:
I'd feel better if there was a teardown that reversed this since `sys.modules` is a global.

Reply:
good point. I've used unittest.patch for this recently, I'll look into using it for this situation.